mgmt2/rpc/jsonrpc/Defs.h

```diff
@@ -45,11 +45,20 @@ class RPCHandlerResponse
 };
 
 struct RPCResponseInfo {
-  RPCResponseInfo(std::optional<std::string> const &id_) : id(id_) {}
+  RPCResponseInfo(std::optional<std::string> const &id_) : id{id_} {}
```

Comment:
Why use `std::optional` with `std::string`? Isn't an empty string the same as not having one?